Robot_Adapter/Convert/ToRobot/Geometry/Curve.cs

```diff
@@ -20,29 +20,35 @@
  * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
  */
 
-using BH.oM.Geometry;
 using BH.Engine.Geometry;
+using BH.oM.Geometry;
 using RobotOM;
+using System;
 using System.Collections.Generic;
 
-namespace BH.Engine.Robot
+namespace BH.Adapter.Robot
 {
     public static partial class Convert
     {
         /***************************************************/
         /****           Public Methods                  ****/
         /***************************************************/
 
-        public static RobotGeoPoint3D FromBHoMGeometry(RobotApplication robotapp, Point point)
+        public static IRobotComponentType ToRobot(ICurve curve)
```

Comment:
Might tweak the name of this method, as it is checking if convert is possible, and if so to what, rather than doing the convert job.